chapter_optimization/minibatch-sgd.md

```diff
@@ -50,6 +50,22 @@ B = torch.randn(256, 256)
 C = torch.randn(256, 256)
 ```
 
+```{.python .input}
+#@tab tensorflow
+%matplotlib inline
+from d2l import tensorflow as d2l
+import tensorflow as tf
+import numpy as np
+
+timer = d2l.Timer()
+# not using tf.xxx here because 
```

Comment:
Use tf.Variable

Reply:
Hey, @terrytangyuan I tried tf.Variables but got the same type of error 'ResourceVariable' object does not support item assignment

Reply:
Use assign() for tf variables

Reply:
Done.